src/main/resources/jshint-shell.js

```diff
@@ -28,6 +28,13 @@
     var OPTIONS_ARG = 4;
 
     var options = JSON.parse(stripJsonComments(args[OPTIONS_ARG]));
+    var globals;
+
+    if (options.globals) {
+        globals = options.globals;
+        delete options.globals;
```

Comment:
I was just following the pattern of the cli.js in the jshint repo. 
https://github.com/jshint/jshint/blob/a79d5dbfd2442dd298874b5e006cccaec16f5fc9/src/cli.js#L282
It doesn't make a functional difference right now.  However, if in the future... JSHint module decides to iterate over options somewhere later on, not deleting `options.globals` could cause an error.